pkg/sentry/fsimpl/fuse/fusefs.go

```diff
@@ -291,7 +342,7 @@ func (i *Inode) Stat(ctx context.Context, fs *vfs.Filesystem, opts vfs.StatOptio
 	// finally be translated into vfs.FilesystemImpl.StatAt() (see
 	// pkg/sentry/syscalls/linux/vfs2/stat.go), resulting in the same flow
 	// as stat(2). Thus GetAttrFlags and Fh variable will never be used in VFS2.
-	req, err := conn.NewRequest(creds, uint32(task.ThreadID()), i.Ino(), linux.FUSE_GETATTR, &in)
+	req, err := conn.NewRequest(creds, uint32(task.ThreadID()), i.NodeID, linux.FUSE_GETATTR, &in)
```

Comment:
Perhaps we need to document this more explicitly? I got stuck on this for a moment, didn't realize they are different